Sources/AWSLambdaRuntimeCore/LambdaHandler.swift

```diff
@@ -83,6 +83,40 @@ extension LambdaHandler {
     }
 }
 
+// MARK: - AsyncLambdaHandler
+
+#if compiler(>=5.5) && $AsyncAwait
+/// Strongly typed, processing protocol for a Lambda that takes a user defined `In` and returns a user defined `Out` async.
+public protocol AsyncLambdaHandler: EventLoopLambdaHandler {
```

Comment:
hmm, an `AsyncLambdaHandler` isn't running the async code on the `EventLoop` at the moment (that's impossible without custom executors), is this proto conformance still right?

Reply:
Yes it is. Those protocols build up on top of each other: